services/dashboard/views/apps/apps.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0
+ * See accompanying LICENSE file.
+ */
+'use strict';
+angular.module('dashboard.apps', [])
+
+  .config(['$routeProvider', function ($routeProvider) {
+    $routeProvider
+      .when('/apps', {
+        label: 'Applications',
+        templateUrl: 'views/apps/apps.html',
+        controller: 'AppsCtrl'
+      });
+  }])
+
+  .controller('AppsCtrl', ['$scope', '$location', 'restapi', function ($scope, $location, restapi) {
+    $scope.view = function (id) {
+      $location.path("/apps/app/" + id);
+    };
+    $scope.kill = function (id) {
+      restapi.killApp(id);
+    };
+
+    $scope.apps = [];
+    restapi.subscribe('/appmasters', $scope,
```

Comment:
Add reference here to the data type AppMastersData. Something like:
//TODO datatype: AppMastersData. Try and do serde using scalajs See #458